### ISSUE TITLE
🐛 Servers dropdown is missing its label

### ISSUE THREAD

**Issue by tatianamac** [COVID19Tracking/website#866]
## Element
[COVID Tracking API Sandbox](https://covidtracking.com/api#status)
Note the above link actually goes to the element above it; this element didn't have an anchor link.
Maybe it should? Separate issue.

<img width="304" alt="select input with https://covidtracking.com and server text above it. Server text is not the label" src="https://user-images.githubusercontent.com/7484549/82087456-fce6ab80-96a4-11ea-8af1-bccb0a8cd6de.png">

## Expected Behaviour
Any form element should have a label so that it is announced with screenreaders. Before any form element, like this dropdown, I'd expect for the screen reader to announce its label.

## Actual Behaviour
The screen reader announces that I'm about to go into a popup button and reads the field, which is the URL. I have no context for what it is unless I can visually see the faux label or I navigate back. This prevents users who are tabbing to interactive elements to know what this is.

## Recommendation
Move the visual `<span>` that is currently outside of the `<label>` element inside of the `<label>` element so it's properly announced. The `<label>` also shouldn't nest the `<select>` element inside of it. It should read:

```
<label>
</label>
<select>
...
```

## Resources
[WebAIM: Form controls
](https://webaim.org/techniques/forms/controls)

## Additional Note
I imagine there's a reason for this but the pattern of having a single-choice select is a weird one to navigate; it allows a lot of interaction that isn't yet necessary. I'm not familiar enough to provide a recommendation but wanted to call attention to it.

**Comment by kevee** [comment]
@friej715 ditto - maybe we should do a support request with Swagger on this?

**Comment by kevee** [comment]
I opened this issue with the Swagger-UI project:
https://github.com/swagger-api/swagger-ui/issues/6015

**Comment by kevee** [comment]
Closing in favor of #935
